pages/blog/[slug].vue

```diff
@@ -0,0 +1,311 @@
+  console.log('In watch preview enabled, newVal, oldVal', newVal, oldVal)
+  page.value = _get(newVal, 'ftvaArticle', {})
+  ftvaRecentPosts.value = _get(newVal, 'ftvaRecentPosts', {})
+})
+
+// COMPUTED
+// Get data for Image or Carousel at top of page
+const parsedImage = computed(() => {
+  return page.value.imageCarousel
+})
+
+// Transform data for Carousel
+const parsedCarouselData = computed(() => {
+  // map image to item, map creditText to credit
+  return parsedImage.value.map((rawItem, index) => {
+    return {
+      item: [{ ...rawItem.image[0], kind: 'image' }], // Carousels on this page are always images, no videos
+      credit: rawItem?.creditText,
+      captionTitle: 'dfdsfs', // TODO do we need these? test without
+      captionText: 'dfsdfsd',
+    }
+  })
+})
+
+// Combine the categories into a String
+const parsedArticleCategories = computed(() => {
+  const categoryList = page.value.articleCategories
+  const categories = []
+
+  for (const category of categoryList) {
+    categories.push(category.title)
+  }
+  return categories.join(', ')
+})
+</script>
+
+<template>
+  <main
+    id="main"
+    class="page page-article-detail"
+  >
+    <div class="one-column">
+      <NavBreadcrumb
+        class="breadcrumb"
+        data-test="breadcrumb"
+        :title="page?.title"
+      />
+
+      <ResponsiveImage
+        v-if="parsedImage && parsedImage.length === 1 && parsedImage[0]?.image && parsedImage[0]?.image?.length === 1"
+        data-test="single-image"
+        :media="parsedImage[0]?.image[0]"
+        :aspect-ratio="43.103"
+      >
+        <template
+          v-if="parsedImage[0]?.creditText"
+          #credit
+        >
+          {{ parsedImage[0]?.creditText }}
+        </template>
+      </ResponsiveImage>
+
+      <div
+        v-else
+        class="lightbox-container"
+      >
+        <FlexibleMediaGalleryNewLightbox
+          data-test="image-carousel"
+          :items="parsedCarouselData"
+        >
+          <template #default="slotProps">
+            <BlockTag
+              data-test="credit-text"
+              :label="parsedCarouselData[slotProps.selectionIndex]?.creditText"
+            />
+          </template>
+        </FlexibleMediaGalleryNewLightbox>
+      </div>
+    </div>
+
+    <div
+      data-test="second-column"
+      class="two-column"
+    >
+      <div class="primary-column top">
+        <SectionWrapper>
+          <CardMeta
+            data-test="text-block"
+            :title="page?.title"
+            :category="parsedArticleCategories"
+          />
+
+          <!-- About the Author -->
+          <RichText
+            v-if="page?.aboutTheAuthor"
+            data-test="aboutTheAuthor"
+            class="acknowledgements"
+            :rich-text-content="page?.aboutTheAuthor"
+          />
+        </SectionWrapper>
+      </div>
+    </div>
+
+    <SectionWrapper>
+      <DividerWayFinder />
+    </SectionWrapper>
+
+    <SectionWrapper>
```

Comment:
flexiblepage blocks don thave to be wrapped in setionwrapper component, check the library website code for examples